source/_docs/tools/quick-bar.markdown

```diff
@@ -3,7 +3,7 @@ title: "Quick Bar"
 description: "Dialog for quickly accessing entities or running commands."
 ---
 
-The "Quick Bar" allows you to quickly look up entities or run commands without needing to navigate away from your current view. (Similar to the "quick open" feature in VS Code, Chrome Developer Tools, etc).
+The "Quick Bar" allows you to quickly lookup entities or run commands without needing to navigate away from your current view. (Similar to the "quick open" feature in VS Code, Chrome Developer Tools, etc.)
```

Comment:
This isn't correct. "Lookup" is a noun ("He performed a lookup"). "Look up" is a verb: "she looked up how to use quick bar".

Reply:
Also (and this was my bad):

```
...current view. (Similar to the "quick open" feature in VS Code, Chrome Developer Tools, etc).
```

should be

```
...current view (similar to the "quick open" feature in VS Code, Chrome Developer Tools, etc).
```